src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -181,6 +190,7 @@ public StreamingResponseBody streamEvents(
                 client.checkScopes(eventType.getReadScopes());
 
                 // validate parameters
+                final TopicRepository topicRepository = timelineService.getTopicRepository(eventType);
```

Comment:
I wanted to make the transition smoother and implement only the part for the changes in topic repository in that way we can test it on live not introducing too much changes. If I am going to create topic repository for each cursor it will touch consumption as well. From the other side it will be simpler to implement consumption later If I have the right api method now, but this is not so many changes.